src/Examples/UnitModels/Operations/skeleton_unit_testing.ipynb

```diff
@@ -0,0 +1,420 @@
+    "m.fs.skeleton.temperature_in = Var(m.fs.time, initialize=298.15, units=pyunits.K)\n",
+    "m.fs.skeleton.pressure_in = Var(m.fs.time, initialize=101, units=pyunits.kPa)\n",
+    "\n",
+    "# output vars for skeleton\n",
+    "m.fs.skeleton.flow_comp_out = Var(m.fs.time, m.fs.skeleton.comp_list, initialize=1.0, units=pyunits.mol/pyunits.s)\n",
+    "m.fs.skeleton.temperature_out = Var(m.fs.time, initialize=298.15, units=pyunits.K)\n",
+    "m.fs.skeleton.pressure_out = Var(m.fs.time, initialize=101, units=pyunits.kPa)\n",
+    "\n",
+    "# Surrogate model equations\n",
+    "\n",
+    "# flow equation\n",
+    "def rule_flow(m, t, i):\n",
+    "    return m.flow_comp_out[t, i] == m.flow_comp_in[t, i]\n",
+    "m.fs.skeleton.eq_flow = Constraint(m.fs.time, m.fs.skeleton.comp_list, rule=rule_flow)\n",
+    "\n",
+    "# note that all terms need to have consistent units - the \"10\" terms could be defined as separate variables with units\n",
+    "m.fs.skeleton.eq_temperature = Constraint(expr=m.fs.skeleton.temperature_out[0]\n",
+    "                                          == m.fs.skeleton.temperature_in[0] + 10*pyunits.K)\n",
+    "m.fs.skeleton.eq_pressure = Constraint(expr=m.fs.skeleton.pressure_out[0]\n",
+    "                                       == m.fs.skeleton.pressure_in[0] - 10*pyunits.kPa)\n",
+    "\n",
+    "# dictionaries relating state properties to custom variables\n",
+    "inlet_dict = {\"flow_mol_comp\": m.fs.skeleton.flow_comp_in,\n",
+    "              \"temperature\": m.fs.skeleton.temperature_in,\n",
+    "              \"pressure\": m.fs.skeleton.pressure_in}\n",
+    "outlet_dict = {\"flow_mol_comp\": m.fs.skeleton.flow_comp_out,\n",
+    "               \"temperature\": m.fs.skeleton.temperature_out,\n",
+    "               \"pressure\": m.fs.skeleton.pressure_out,}\n",
+    "\n",
+    "m.fs.skeleton.add_ports(name=\"inlet\", member_dict=inlet_dict)\n",
+    "m.fs.skeleton.add_ports(name=\"outlet\", member_dict=outlet_dict)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's see how many degrees of freedom the flowsheet has:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "4\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(degrees_of_freedom(m))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "By specifying four input values, we obtain a square problem and can initialize and solve the flowsheet similar to other IDAES unit models. While importing IDAES unit models requires fully consistent physical unit sets, the Skeleton model allows users to only include a few specific equations and easily allows custom physical units as well. As shown below, the model only requires unit consistency within equations as no other unit, block or property modules are called:"
```

Comment:
1. This paragraph seems to cover two separate topics, and I think it should be broken into two with more discussion. The first part deals with degrees of freedom, and the second part with consistency of "units" (see my third point).

2.I think you can say more about degrees of freedom, such as that we would expect 4 degrees of freedom (referring back to the introduction of the model as appropriate). From that, explicitly call out the 4 things we intend to fix to make the problem square.

3. I am not sure what the last part of this is trying to say - how does "importing IDAES unit models requires fully consistent physical unit sets", and how does the `SkeletonModel` address this?

Notably, I think you are using the term "units" in multiple senses in this one paragraph. The first part is saying the "units don't need to be consistent", and then the final sentence says "he model only requires unit consistency within equations" which would appear to be contradictory. What I think you mean is that `SkeletonModels` do not require consistent thermophysical properties, but consistency of units of measure must still be maintained. This needs to be clarified, and you should choose your words carefully.